backend/protocol_rpc/endpoints.py

```
@@ -415,6 +446,197 @@ def count_validators(validators_registry: ValidatorsRegistry) -> int:
+    )
+
+    return deploy_tx.from_address if deploy_tx else None
+
+
+def get_contract_nonce(session: Session, contract_address: str) -> int:
+    """Get contract nonce (tx count TO this contract) for upgrade signatures."""
+    from backend.database_handler.models import Transactions
+
+    try:
+        checksum_address = eth_utils.to_checksum_address(contract_address)
+    except:
```

Ruff (0.14.8)

Do not use bare `except`

(E722)